### DISCUSSION TITLE
Chips Paste (Comma separated) is not working

### DISCUSSION THREAD

**Discussion by HSnicolascerrejon** [primefaces/community#1145] [PrimeFaces] [answered]
Chips Paste (Comma separated) is not working. You can check it in the showcase.

Regards, Nico.

**Comment by melloware** [comment]
Please open a GitHub bug ticket.  https://github.com/primefaces/primefaces/issues

**Reply by HSnicolascerrejon** [reply, marked as answer]
Done: https://github.com/primefaces/primefaces/issues/11407

**Reply by melloware** [reply]
Fixed for 13.0.6
